server/newTransactionSubmitter.js

```diff
@@ -0,0 +1,48 @@
+require('isomorphic-fetch')
+
+async function request(url, method = 'get', body = null, headers = {}) {
+  const res = await fetch(url, {
+    method,
+    headers,
+    body,
+  })
+
+  return res.json()
+}
+
+module.exports = function(app, env) {
+  // eslint-disable-next-line consistent-return
+  app.post('/api/txs/submit', async (req, res) => {
+    let txHash
+    let txBody
+    try {
+      txHash = req.body.txHash
+      txBody = req.body.txBody
+      if (!txHash || !txBody) throw new Error('bad format')
+    } catch (err) {
+      return res.json({
+        Left: 'Bad request body',
+      })
+    }
+
+    const signedBody = {
+      signedTx: Buffer.from(txBody, 'hex').toString('base64'),
+    }
+
+    try {
+      const fetched = await request(
+        `${process.env.ADALITE_BLOCKCHAIN_EXPLORER_URL}/api/v2/txs/signed`,
+        'POST',
+        JSON.stringify(signedBody),
+        {
+          'Content-Type': 'application/json; charset=utf-8',
+        }
+      )
+      return res.json(fetched)
+    } catch (err) {
+      return res.json({
+        Left: `Error: ${err}`,
```

Comment:
returning the error message as-is may be too revealing. I would prefer logging it and returning just "An unexpected error has happened" - same as in the old transaction submitter.